cohesity/src/agents/special/agent_cohesity

```diff
@@ -48,8 +48,8 @@ def get_header():
     try:
         response = requests.post(
             url, data=creds, headers=header, verify=False)
-    except APIException as exp:
```

Comment:
@sritd I think we should catch both exceptions here: except (APIException, OSError) as exp. Wondering in which case we have an OSError?

Reply:
APIException is not defined. Missing import?

Reply:
OSError is thrown, if connection can not be established.

Reply:
OK, APIException is left from the SDK (which is not used anymore and therefore wrong, thank you for removing). But OSError is not a requests exception, just the base for the base Exceptions in the Requests module: https://docs.python-requests.org/en/master/_modules/requests/exceptions/.

So general would be to catch RequestException or ConnectionError?

Reply:
That is not enough. OSError is thrown, if DNS can not be resolved or connection is not possible. That is not done by requests module. That is done by socket. I think, this is correct.